scrypto-derive/src/import.rs

```diff
@@ -343,13 +343,13 @@ fn get_native_type(ty: &SchemaType) -> Result<(Type, Vec<Item>)> {
             structs.extend(s);
             parse_quote! { ::scrypto::component::KeyValueStore<#k, #v> }
         }
-        SchemaType::Bucket => parse_quote! {::scrypto::engine_lib::resource::Bucket },
-        SchemaType::Proof => parse_quote! { ::scrypto::engine_lib::resource::Proof},
-        SchemaType::Vault => parse_quote! { ::scrypto::engine_lib::resource::Vault},
+        SchemaType::Bucket => parse_quote! {::scrypto::model::Bucket },
+        SchemaType::Proof => parse_quote! { ::scrypto::model::Proof},
+        SchemaType::Vault => parse_quote! { ::scrypto::model::Vault},
         SchemaType::Expression => parse_quote! {::scrypto::runtime::Expression },
-        SchemaType::Blob => parse_quote! { ::scrypto::engine_lib::crypto::Blob},
+        SchemaType::Blob => parse_quote! { ::scrypto::model::Blob},
         SchemaType::NonFungibleAddress => {
-            parse_quote! { ::scrypto::engine_lib::resource::NonFungibleAddress}
+            parse_quote! { ::scrypto::model::NonFungibleAddress}
         }
         SchemaType::Hash => parse_quote! { ::scrypto::crypto::Hash},
```

Comment:
I don't believe `::scrypto::crypto::` hash exists either...

Perhaps we should create it as:
```
/// Scrypto crypto types.
pub mod crypto {
    pub use radix_engine_interface::crypto::*;
}
```

Reply:
Nice catch!